ext/priority_sampling/priority_sampling.c

```diff
@@ -123,36 +123,38 @@ static ddtrace_rule_result dd_match_rules(ddtrace_span_data *span, bool eval_roo
     }
 
     zval *rule;
-    ZEND_HASH_FOREACH_VAL(get_DD_TRACE_SAMPLING_RULES(), rule) {
-        if (++index >= skip_at) {
-            break;
-        }
-
-        if (Z_TYPE_P(rule) != IS_ARRAY) {
-            continue;
-        }
+    if (!get_global_DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED()) {  // APPSEC_STANDALONE enabled, override sampling rules to be empty
```

Comment:
This just wrap the code within `if (!get_global_DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED()) {`